models/metric.txt

```diff
@@ -1,66 +0,0 @@
-service/server-service:
```

Comment:
also, this file was my attempt at understanding the relationships between the models

IMO, @altsang @ritch @seanbrookes @raymondfeng Loopback/strong-arc ***desperately*** needs a model relationship viewer that graphically displays the relationships between models. Figuring it out from the json is pretty hard, I always end up drawing a picture on paper, or writing a text file summary like this.